CogniteSdk.Types/Annotations/AnnotationCreate.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+using CogniteSdk.Types.Common;
+
+namespace CogniteSdk
+{
+    /// <summary>
+    /// The Annotation create class
+    /// </summary>
+    public class AnnotationCreate: AnnotationCommon
+    {
+        /// <summary>
+        /// The annotation information. The format of this object is decided by and validated against the 'AnnotationType' attribute.
+        /// </summary>
+        public BoundingVolume Data {get; set;}
+
+
```

Comment:
Double newline

Reply:
fixed in [8a6f4ca](https://github.com/cognitedata/cognite-sdk-dotnet/pull/287/commits/8a6f4ca65ad825bfa2d79a3230cc4ddc834da8e1)